docs/content/guides/dagster-pipes.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Dagster Pipes | Dagster Docs
+description: "Dagster Pipes provides a protocol between the orchestration environment (Dagster) and external execution (ex: Databricks) and a toolkit for building implementations of that protocol."
+---
+
+# Dagster Pipes
+
+Dagster Pipes is a toolkit for building integrations between Dagster and external execution environments. It standardizes the process of passing parameters, injecting context information, ingesting logs, and collecting metadata all while remaining agnostic to how remote computations are launched in those environments. This enables the separation of orchestration and business logic in the Dagster ecosystem.
+
+It also smooths the process of incorporating pre-existing code, business logic, and execution environments into Dagster. Using Dagster Pipes, you add just a few lines to your code and it becomes seamlessly invocable by the orchestrator. In turn, you can stream logs and metadata back to Dagster so you can leverage its observability, lineage, cataloging, and debugging capabilities.
```

Comment:
I would remove the word `just` here, even a few lines of code can be more difficult than it appears. 
"it becomes seamlessly invocable by the orchestrator" I find this a little confusing as well, maybe we can simplify this whole sentence:

> With Dagster Pipes and a few lines of code, you can execute your code through the orchestrator. In turn, you can stream logs ... "